<launch>
  <node pkg="project1" type="computing" name="velocity_comp" />
  <node pkg="rosbag" type = "play" name="rosbag" args="/home/feder/Desktop/Programmazione/robotics/src/project1/src/bag1.bag" />
</launch>
